### PR TITLE
fix(personal-representative): Gitbook and docker setup update

### PR DESCRIPTION
**Pull request by FridrikMProgramm**
Gitbook update
Also adding dockerizing to workspace settings

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/services/personal-representative-public/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+You can serve this service locally by running:
+
+```bash
+yarn start services-personal-representative-public
+```
+
+Api open api specs will now be accessible at
+
+```bash
+http://localhost:3378/swagger/#/default
```
Comment (saevarma) [inline apps/services/personal-representative-public/README.md:31]:
```suggestion
http://localhost:3378/swagger
```

File: apps/services/personal-representative/README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ yarn start services-personal-representative
 Api open api specs will now be accessible at
 
 ```bash
-http://localhost:3333/swagger/#/default
+http://localhost:3378/swagger/#/default
```
Comment (saevarma) [inline apps/services/personal-representative/README.md:31]:
```suggestion
http://localhost:3378/swagger
```

**Review by saevarma — CHANGES_REQUESTED** [review]
This PR includes 239 commits, I think there is something strange with the history of this branch?

**Comment by FridrikMProgramm** [comment]
> This PR includes 239 commits, I think there is something strange with the history of this branch?
I reused a deleted PR, might have something to do with it. But it only seems to affect the "history" not the outcome.

**Review by fjandakornid — APPROVED** [review]
LGTM

**Pull request closed without merging**